.jenkins/caffe2/test.sh

```diff
@@ -62,7 +62,12 @@ for test in $(find "${INSTALL_PREFIX}/test" -executable -type f); do
     */mkl_utils_test|*/aten/integer_divider_test)
       continue
       ;;
-    *)
+    */scalar_tensor_test)
+	  if [[ "$BUILD_ENVIRONMENT" == *rocm* ]]; then
+		continue
+	  fi
```

Comment:
Need to add an `else` branch to run the test in non-rocm builds?

Reply:
@bddppq I dont think so, since the logic here is " 'continue' when bad test occured".
If it is not a bad test we can just let it go.

Reply:
But your current version is skipping this bad test for all build variants. ("continue" means go to next iteration, not go to next `case`).

Reply:
T T I see...